staging/velero/charts/minio/templates/cleanup-job.yaml

```diff
@@ -44,6 +44,12 @@ spec:
                 kubectl delete minioinstances -n {{ .Release.Namespace }} minio
                 kubectl delete statefulset -n {{ .Release.Namespace }} --ignore-not-found minio
               fi
+              # In case rolling update is not possible, which does
+              # happen in many minio releases, we'll need to delete
+              # the statefulset.
+              {{- if .Values.statefulSetForceCleanup }}
+              kubectl delete statefulset -n {{ .Release.Namespace }} --ignore-not-found {{ template "minio.statefulset.name" . }}
```

Comment:
What happens to data when this option is used?
Is it safe to always pass in `true` here?

Reply:
@dkoshkin data won't be lost because pvc won't be deleted, and the new statefulset will pick up the left pvc. And yes, it's always safe to pass `true` here.